Sources/PostgresNIO/New/Data/Range+PostgresCodable.swift

```diff
@@ -0,0 +1,104 @@
+import NIOCore
+
+extension RangeExpression where Bound: PostgresRangeBound {
+    public static var psqlFormat: PostgresFormat { return .binary }
+}
```

Comment:
Can we link the postgres docs here, why this is the case?

Reply:
I'm actually not sure, though I did notice that all the other implementations of `PostgresEncodable` return `.binary` as the value for their `psqlFormat`.

Is there a scenario where we want to enable the option of `.text` even though `.binary` works?